### ISSUE TITLE
Sorting in History Window Only Sorts Current Page

### ISSUE THREAD

**Issue by uyaem** [harlam357/hfm-net#333]
I've finally processed more than 1000 work units that were monitored by HFM, so the history window pages.

As the title says, if I e.g. sort by "Finished (UTC)" descending, it will only sort the items on the current page. So the top row will display the last completed of the first $page_size items, not the last completed overall. That has fooled me a few times already. :)

Is this intended, or a side-effect of the sorting implementation (without looking: sorting the grid, not the data source)?

**Comment by harlam357** [comment]
Precisely, sorting the grid, not the data source.  The generated queries have no provision for sorting at the data source.  That's a good thought though.  Perhaps adding an additional section for order by in the query dialog.

I also sort by "Finished (UTC)" descending and to see the latest I always just click the "last page" button here.

![image](https://user-images.githubusercontent.com/13512136/90710624-635f0500-e264-11ea-9047-e37532cb4f03.png)

Perhaps there's also an opportunity for an option to start on the last page, not the first.

**Comment by PantherX** [comment]
Ah, thanks for reporting this @uyaem as I have spent the last few weeks in circles and was wondering what was happening. I accidentally discovered the solution of @harlam357 after systematically clicking buttons at random until I saw what I was looking for 😆
